shared-helpers/__tests__/localeRoute.test.ts

```diff
@@ -0,0 +1,26 @@
+import { cleanup } from "@testing-library/react"
+import { lRoute } from "../src/localeRoute"
+import { addTranslation } from "@bloom-housing/ui-components"
+import spanishTranslations from "@bloom-housing/ui-components/src/locales/es.json"
+import chineseTranslations from "@bloom-housing/ui-components/src/locales/zh.json"
+import vietnameseTranslations from "@bloom-housing/ui-components/src/locales/vi.json"
+import generalTranslations from "@bloom-housing/ui-components/src/locales/general.json"
+
+afterEach(cleanup)
+
+describe("localeRoute helper", () => {
```

Comment:
I think we can safely do away with `localeRoute` at this point…seems like a holdover from pre Next v10 days when we had to roll our own localized routing.